onecodex/models/collection.py

```diff
@@ -92,33 +223,14 @@ def __init__(self, *args, **kwargs):
             )
             kwargs["metric"] = kwargs.pop("field")
 
-        if len(args) == 2 and isinstance(args[0], list) and issubclass(args[1], OneCodexBase):
-            self._resource_list_constructor(*args, **kwargs)
-        else:
-            self._sample_collection_constructor(*args, **kwargs)
-
-    def _resource_list_constructor(
-        self,
-        _resource,
-        oc_model,
-        skip_missing=True,
-        metric="auto",
-        include_host=False,
-        job=None,
-    ):
-        self._kwargs = {
-            "skip_missing": skip_missing,
-            "metric": metric,
-            "include_host": include_host,
-            "job": job,
-        }
-        super(SampleCollection, self).__init__(_resource, oc_model, **self._kwargs)
+        self._sample_collection_constructor(*args, **kwargs)
 
     def _sample_collection_constructor(
```

Comment:
Now that `_resource_list_constructor()` was removed, does it make sense to inline `_sample_collection_constructor()` into `__init__()`?